semgrep/setup.py

```diff
@@ -108,6 +108,7 @@ def find_executable(env_name, exec_name):
         "tqdm>=4.46.1",
         "packaging>=20.4",
         "jsonschema~=3.2.0",
+        "wcmatch>=8.2",
```

Comment:
```suggestion
        "wcmatch==8.2",
```
Can we pin to a specific version of wcmatch for now. A change in file targeting behavior would have an outsized impact on semgrep behavior.